File: src/main/java/org/game/Easy.java
package org.game;

import javax.swing.*;

/**
 * 8 x 8, 16 mines
 */

public final class Easy implements GameType {
    private Easy() { }

    /**
     * 8 x 8, 16 mines
     * @param name name of player
     */

    public static final void start(final String name, final String password) {
        JOptionPane.showMessageDialog(
                null,
                "Good luck, " + name + "!",
                "Greetings",
                JOptionPane.INFORMATION_MESSAGE
        );

        new Game(8, 8, 16, name, password, new Easy()).start();
    }
}

File: src/main/kotlin/org/game/Easy.kt
package org.game

import javax.swing.JOptionPane

/**
 * 8 x 8, 16 mines
 */

internal class Easy : GameType {
    internal companion object {

        /**
         * 8 x 8, 16 mines
         * @param name name of player
         */

        @JvmStatic
        internal fun start(name: String, password: String) {
            JOptionPane.showMessageDialog(
                null,
                "Good luck, $name!",
                "Greetings",
                JOptionPane.INFORMATION_MESSAGE
            )

            Game(8, 8, 16, name, password, Easy()).start()
        }
    }
}